package.json

```diff
@@ -21,7 +21,7 @@
     "aes-js": "^3.1.2",
     "bfx-api-node-core": "^1.5.1",
     "bfx-api-node-rest": "^4.1.2",
-    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.5.0",
+    "bfx-hf-algo": "git+https://github.com/tarcisiozf/bfx-hf-algo.git#auth-response",
```

Comment:
wait for merge

Reply:
@tarcisiozf https://github.com/bitfinexcom/bfx-hf-algo/releases/tag/v4.5.1